code/game/machinery/alarm.dm

```diff
@@ -163,6 +163,21 @@
 	target_temperature = T0C+20
 	scrubbers_gases = list("oxygen" = 1, "nitrogen" = 1, "carbon_dioxide" = 1, "plasma" = 0, "n2o" = 0)
 
+/datum/airalarm_preset/vacuum
+	name = "Vacuum"
+	desc = "For rooms to be kept under vacuum"
+	core = TRUE
+	oxygen = list(-1, -1, 0.5, 1)
+	nitrogen = list(-1, -1, 0.5, 1)
+	carbon_dioxide = list(-1, -1, 0.5, 1)
+	plasma = list(-1, -1, 0.5, 1)
+	n2o = list(-1, -1, 0.5, 1)
+	other = list(-1, -1, 0.5, 1)
+	pressure = list(-1, -1, ONE_ATMOSPHERE*0.01, ONE_ATMOSPHERE*0.05)
+	temperature = list(-1, -1, -1, -1)
+	target_temperature = T0C+20
+	scrubbers_gases = list("oxygen" = 1, "nitrogen" = 1, "carbon_dioxide" = 1, "plasma" = 1, "n2o" = 0)
+
```

Comment:
Why.  Just set up a floor vent set to input.

Reply:
The air alarm will throw a fit if the pressure in the room drops too low without this preset.